Traceback (most recent call last):
  File "2_train.py", line 20, in <module>
    import utils.import_envs  # pytype: disable=import-error
  File "/home/pierre/HDD/0_Complearn/1_learning/0_Reinforcement_learning/18_replab/pierreExeter_github/stable-baselines_zoo/utils/__init__.py", line 1, in <module>
    from .utils import make_env, ALGOS, linear_schedule, create_test_env,\
  File "/home/pierre/HDD/0_Complearn/1_learning/0_Reinforcement_learning/18_replab/pierreExeter_github/stable-baselines_zoo/utils/utils.py", line 18, in <module>
    from stable_baselines.deepq.policies import FeedForwardPolicy
  File "/home/pierre/HDD/0_Complearn/1_learning/0_Reinforcement_learning/18_replab/pierreExeter_github/stable-baselines/stable_baselines/__init__.py", line 3, in <module>
    from stable_baselines.a2c import A2C
  File "/home/pierre/HDD/0_Complearn/1_learning/0_Reinforcement_learning/18_replab/pierreExeter_github/stable-baselines/stable_baselines/a2c/__init__.py", line 1, in <module>
    from stable_baselines.a2c.a2c import A2C
  File "/home/pierre/HDD/0_Complearn/1_learning/0_Reinforcement_learning/18_replab/pierreExeter_github/stable-baselines/stable_baselines/a2c/a2c.py", line 5, in <module>
    import tensorflow as tf
  File "/home/pierre/bin/anaconda3/envs/SB_widowx/lib/python3.6/site-packages/tensorflow/__init__.py", line 47, in <module>
    from tensorflow._api.v1 import image
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 951, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 894, in _find_spec
  File "<frozen importlib._bootstrap_external>", line 1157, in find_spec
  File "<frozen importlib._bootstrap_external>", line 1129, in _get_spec
  File "<frozen importlib._bootstrap_external>", line 1252, in find_spec
KeyboardInterrupt
